padd.sh

```diff
@@ -87,16 +83,13 @@ mega_status_unknown="${check_box_question} Unable to determine Pi-hole status."
 # TINY STATUS
 tiny_status_ok="${check_box_good} System is healthy."
 tiny_status_update="${check_box_info} Updates are available."
```

Comment:
Is this warning supposed to be eliminated?

Reply:
I think we are talking about the same reason (enabling SC2034 and SC2154). Do we?